pubsub/rocketmq/rocketmq.go

```diff
@@ -146,208 +273,198 @@ func (r *rocketMQ) Features() []pubsub.Feature {
 	return nil
 }
 
-func (r *rocketMQ) Publish(req *pubsub.PublishRequest) error {
-	r.logger.Debugf("rocketmq publish topic:%s with data:%v", req.Topic, req.Data)
-	msg := newRocketMQMessage(req)
-
-	publishBo := backoff.NewExponentialBackOff()
-	publishBo.InitialInterval = 100 * time.Millisecond
-	bo := backoff.WithMaxRetries(publishBo, 3)
-	bo = backoff.WithContext(bo, r.ctx)
-	return retry.NotifyRecover(
```

Comment:
Why did you remove the backoff?

Reply:
The RocketMQ Client will retry if it fails. The retry times are the values of metadata.Retries.
We don't need to retry here.